packages/angular/projects/angular/src/internal/rxjs/use-store-selector.ts

```diff
@@ -0,0 +1,20 @@
+import { Selector, StoreState, UnknownRecord } from '@devextreme/core';
+import { finalize, Observable, startWith } from 'rxjs';
+
+export function useStoreSelector<
```

Comment:
Minor suggestion, Mind sync, Not required to response
useStoreSelector is just a transparent wrapper of core observable into the rx observable
May be we can directly use rx observale as a parameter in the createStore and doesnt need such useStoreSelector